README.md

```diff
@@ -45,6 +45,14 @@ cops together with the standard cops.
 rubocop --require rubocop-rspec
 ```
 
+Alternatively, use the following array notation when specifying multiple extensions.
```

Comment:
I believe this should go to the previous section, after line 38.

Reply:
thank you, surely after line 38 is good

Reply:
fixed it!

https://github.com/rubocop-hq/rubocop-rspec/pull/783/commits/1eede967354d99c0ffaf1ca57d7fbfc0c9a1d8cc